libbpf-tools/offcputime.c

```diff
@@ -154,6 +161,23 @@ static error_t parse_arg(int key, char *arg, struct argp_state *state)
 			fprintf(stderr, "Invalid task state: %s\n", arg);
 			argp_usage(state);
 		}
+		break;
+	case 'P':
+		env.unwind = true;
+		break;
+	case OPT_SAMPLE_STACK_SIZE:
```

Comment:
> I think we should make stack size as part of the input for -P. If not specified, you could use 128 bytes.

offcputime has a positional argument for the duration, so -P cannot be set to OPTION_ARG_OPTIONAL.
Therefore, a separate option for stack size has been added.